tasks/setup.py

```diff
@@ -23,6 +23,8 @@ def setup(_: Context, config_path: Optional[str] = None, copy_to_clipboard: Opti
     info("🤖 Install Pulumi")
     if is_windows():
         os.system("winget install pulumi")
+    elif is_linux():
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Thank you !